### ISSUE TITLE
Spawner with wrong mod type

### ISSUE THREAD

**Issue by Eatham532** [YUNG-GANG/YUNGs-Better-Strongholds#69]
I'm not too sure about this. But a zombie in this mob spawner just seems wrong. I think it should probably be a spider because otherwise, the zombie would get stuck in the webs. Also, a spider just seems right with there being webs everywhere. 

![2022-10-04_21 49 15](https://user-images.githubusercontent.com/78714349/193945761-21518b94-b4df-4d8a-9341-56aa40cf7e7a.png)
The photo is of the spawner in the big library

Version: 1.19.2, V3.2.1

**Comment by yungnickyoung** [comment]
It should definitely be a spider spawner. Fabric or Forge? And are you using other mods that may affect spawners?

**Comment by Eatham532** [comment]
Fabric sorry, forgot to mention that. All the other issues I opened are fabric.

**Comment by Eatham532** [comment]
No i was just using your mods

**Comment by yungnickyoung** [comment]
Yeah, looks like a few of them are set to zombies instead of spiders. Will fix that in a future release
